starwars/src/components/NextButton.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './Buttons.css';
+
+const NextButton = (props) => {
+  return (
+    <button type="submit" onClick={props.nextPage}>Next</button>
```

Comment:
The next button works great! I would try to make it scroll to the top of the page somehow though? So every time you click on the next or prev button it should take you to the top of the page. 😊